linux-anvil-cos7-x86_64/Dockerfile

```diff
@@ -10,10 +10,10 @@ ENV CENTOS_VER=${CENTOS_VER}
 ENV LANG en_US.UTF-8
 
 # Add qemu in here so that we can use this image on regular linux hosts with qemu user installed
-ADD qemu-aarch64-static /usr/bin/qemu-aarch64-static
-ADD qemu-arm-static /usr/bin/qemu-arm-static
-ADD qemu-ppc64le-static /usr/bin/qemu-ppc64le-static
-ADD qemu-s390x-static /usr/bin/qemu-s390x-static
+# ADD qemu-aarch64-static /usr/bin/qemu-aarch64-static
+# ADD qemu-arm-static /usr/bin/qemu-arm-static
+# ADD qemu-ppc64le-static /usr/bin/qemu-ppc64le-static
+# ADD qemu-s390x-static /usr/bin/qemu-s390x-static
```

Comment:
FYI, this block caused the same uid/gid issues on those commands, so I had to comment the lines out.